src/main/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientImpl.java

```diff
@@ -246,4 +282,23 @@ private Mono<ClientResponse> mapErrorResponse(ClientResponse clientResponse) {
             .error(new ChatbotResponseException(clientResponse.statusCode(), errorDetails,
                 clientResponse.headers())));
   }
+
+  private String getCurrentApiVersion() throws IOException {
+    Properties properties;
+    InputStream is = getClass().getClassLoader()
+        .getResourceAsStream("properties-from-pom.properties");
```

Comment:
Can you share the example content of `properties-from-pom.properties`?

Reply:
```
#Properties
#Tue Dec 20 10:59:39 PST 2022
spring-boot-starter-webflux-version=2.5.12
swagger-core-version=1.6.2
spring-reactor-test-version=3.2.3.RELEASE
openapi-generator-version=5.4.0
einstein-bot-openapi-codegen-version=5.4.4
jackson-annotations-version=2.13.4
jackson-databind-nullable-version=0.2.1
maven-javadoc-plugin.version=3.2.0
base-sdk-package=com.salesforce.einsteinbot.sdk
maven-source-plugin.version=3.2.1
jackson-version=2.13.4.2
project.build.sourceEncoding=UTF-8
maven.compiler.source=1.8
spring-reactor-core-version=3.4.5
junit-jupiter-version=5.5.2
java.version=1.8.0_332
runtime-api-spec-url=https\://runtime-api-na-west.prod.chatbots.sfdc.sh/docs/specs/v5_0_0_api_specs.yml
maven.compiler.target=1.8
maven-release-plugin.version=3.0.0-M1
javax-annotation-version=1.3.2
api-spec-yaml-file=v5_0_0_api_specs.yml

```